ivy/data_classes/container/base.py

```diff
@@ -1218,6 +1222,66 @@ def cont_from_disk_as_json(json_filepath, ivyh=None):
         with open(json_filepath) as json_data_file:
             return ivy.Container(json.load(json_data_file), ivyh=ivyh)
 
+    @staticmethod
+    def cont_from_disk_as_flax(ckpt_filepath, ivyh=None):
+        """Load container object from disk at the specified flax ckpt filepath.
+
+        Parameters
+        ----------
+        ckpt_filepath
+            Filepath where the container object is saved to disk.
+        ivyh
+            Handle to ivy module to use for the calculations. Default is ``None``, which
+            results in the global ivy.
+
+        Returns
+        -------
+            Container loaded from disk
+        """
+        from flax.training import checkpoints
+
+        ckpt = checkpoints.restore_checkpoint(
+            ckpt_dir="/workspaces/ivy/load-save-modules/jax_weights", target=None
```

Comment:
Let's replace the hardcoded absolute path here and use the `ckpt_filepath` instead